defaults/main.yml

```diff
@@ -679,6 +730,27 @@ gitlab_pages_dest: '{{ gitlab_src_path + "/" + gitlab_pages_repo.split("://")[1]
 gitlab_pages_checkout: '{{ gitlab_app_root_path + "/gitlab-pages" }}'
                                                                    # ]]]
                                                                    # ]]]
+# Gitaly source code [[[
+# --------------------------------
+
+# .. envvar:: gitaly_repo [[[
+#
+# URL of the Gitaly repository.
+gitaly_repo: 'https://gitlab.com/gitlab-org/gitaly.git'
```

Comment:
This and other `gitaly_` variables need to use the `gitlab__` variable namespace. Please rename them to `gitlab__gitaly_*`.